doc/get-started/sql-pipeline.rst

```diff
@@ -26,9 +26,40 @@ Or run it locally:
 .. code-block:: console
 
     ploomber examples --name templates/spec-api-sql
+  
+  
+You can also refer to https://github.com/ploomber/projects/blob/master/templates/spec-api-sql/README.ipynb for more information on using SQL scripts to manipulate data in a database, dump a table, and plot it with Python.
```

Comment:
please add the readme link using .rst link format:

```rst
`Link text <link URL>`_
```

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html